docs/en_US/guides/faq.md

```diff
@@ -111,12 +111,13 @@ While this can be easily figured out by checking <router-link to="/get-started">
 - **A7 iPads**: iOS 12.5.7 (via Odysseyra1n or Chimera)
 - **A7/A8 iPhones**: iOS 12.5.7 (via Odysseyra1n or Chimera)
 - **iPod Touch (6th Generation)**: iOS 12.5.7 (via Odysseyra1n or Chimera)
-- **A8(X) iPads**: iPadOS 15.7.4 (via palera1n)
-- **iPod Touch (7th Generation)**: iOS 15.7.4 (via palera1n)
-- **A9/A10 iPhones**: iOS 15.7.4 (via palera1n)
-- **A9(X)-A10(X) iPads**: iPadOS 16.3.1 (via palera1n)
-- **A11 iPhones**: iOS 16.3.1 (via palera1n)
-- **A12-A15 iPhones**: iOS 15.4.1 (via Fugu15 Max)
-- **A12-A15/M1 iPads**: iPadOS 15.4.1 (via Fugu15 Max)
-
-Note that A12 and newer devices running iOS 15.5 or newer do not have a jailbreak, which also inherently means no M2 or A16 device has a jailbreak currently.
\ No newline at end of file
+- **A8(X) iPads**: iPadOS 15.7.9 (via palera1n)
+- **iPod Touch (7th Generation)**: iOS 15.7.9 (via palera1n)
+- **A9/A10 iPhones**: iOS 15.7.9 (via palera1n)
+- **A9(X) iPads**: iPadOS 16.5 (via palera1n)
+- **A10(X) iPads**: iPadOS 17.0 (via palera1n, requires nightly build for now)
```

Comment:
Don't refer to nightly build stuff on this page -> just list off 16.7 until palera1n is updated

Reply:
aight